.jules/bolt.md

```
@@ -0,0 +1,3 @@
+## 2024-11-20 - Fast RMS Energy Calculation using np.vdot
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)